README.md

```diff
@@ -17,7 +17,7 @@ If you are using Maven with [BOM][libraries-bom], add this to your pom.xml file
     <dependency>
       <groupId>com.google.cloud</groupId>
       <artifactId>libraries-bom</artifactId>
-      <version>4.2.0</version>
+      <version>4.3.0</version>
```

Comment:
@chingor13 

What triggered this change?

Reply:
It should have been https://github.com/googleapis/java-video-intelligence/commit/29c5f91d272ec63e595200e9a269dbfaa60a1fef